plugins/cmd/chainlink-starknet/main.go

```diff
@@ -83,7 +83,11 @@ func (c *pluginRelayer) NewRelayer(ctx context.Context, config string, loopKs lo
 		return nil, fmt.Errorf("failed to decode config toml: %w", err)
 	}
 
-	ksAdapter := keystore.NewStarkNetKeystoreAdapter(loopKs)
+	loopsigner, ok := (loopKs).(*keystore.StarknetLooppSigner)
```

Comment:
This _cannot_ be enforced with both runtime modes because we dial a `loop.Keystore` client or grpc. Please follow the structure established by Solana.